File: C/problem solving/Program40.c
// Display the number from n to 0 where n value take from user with the help of iteration.

#include<stdio.h>

void Display(int iValue)
{
    int iCnt = 0 ;

    iCnt = iValue ;
    while(iCnt >= 0)
    {
        printf("%d \n",iCnt);
        iCnt-- ;
    }
}

int main()
{
    int iFrequency = 0 ;

    printf("Enter the frequency : \n ");
    scanf("%d",&iFrequency);

    Display(iFrequency);

}

/*
    OUTPUT

    Enter the frequency :
    5
    5
    4
    3
    2
    1
    0


    Enter the frequency :
    10
    10
    9
    8
    7
    6
    5
    4
    3
    2
    1
    0


    Enter the frequency :
    12
    12
    11
    10
    9
    8
    7
    6
    5
    4
    3
    2
    1
    0

*/

File: JAVA/Problem solving/Program40.java
/*
Accept N numbers from user and return difference between summation of even elements and summation of odd elements.
Input: N: 6
Elements:85 66 3 80 93 88
Output: 53 (234 - 181)
*/

import java.util.Scanner ;

class Program40
{
    public static void display(int[] iValue)
    {
        int i = 0, iSumEven = 0, iSumOdd = 0 ;

        for(i = 0; i < iValue.length; i++)
        {
            System.out.print(iValue[i]+"\t");
            if(iValue[i] % 2 == 0)
            {
                iSumEven = iSumEven + iValue[i];
            }
            else if(iValue[i] % 2 != 0)
            {
                iSumOdd = iSumOdd + iValue[i];
            }
            else
            {
                //
            }
        }
        System.out.println("\nOUTPUT: "+(iSumEven-iSumOdd));
        System.out.println("Sum of even number is: "+ iSumEven + " Sum of odd number is: "+ iSumOdd);
    }

    public static void main(String[] arg)
    {
        int iNum[], iSize = 0, i = 0 ;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the size: ");
        iSize = sobj.nextInt();

        iNum = new int[iSize];

        System.out.println("Enter "+iSize+" numbers: ");
        for(i = 0; i < iSize; i++)
        {
            iNum[i] = sobj.nextInt();
        }

        display(iNum);
    }
}